static/gsApp/views/subscriptionPage/subscriptionHeader.tsx

```diff
@@ -100,21 +105,27 @@ function SubscriptionHeader(props: Props) {
         data-test-id="subscription-page"
         title={t('Subscription')}
         tabs={
-          <NavTabs underlined>
-            {tabConfig.reduce((acc, {key, name, show}) => {
-              if (show(organization, isDisabled, subscription)) {
-                acc.push(
-                  <ListLink
-                    key={key}
-                    to={`/settings/${organization.slug}/billing/${key}/`}
-                  >
-                    {name}
-                  </ListLink>
-                );
-              }
-              return acc;
-            }, [] as React.ReactNode[])}
-          </NavTabs>
+          <TabsContainer>
+            <Tabs value={activeTab.key}>
+              <TabList>
+                {tabConfig.reduce<any>((acc, {key, name, show}) => {
```

Comment:
I don’t think we should be using `reduce` for mapping over arrays, especially if we have to add type assertions like `<any>` to them. What we want here is a `flatMap`:

```
{tabConfig.flatMap(({key, name, show}) =>
  show(organization, isDisabled, subscription) ? (
    [<ListLink key={key} to={`/settings/${organization.slug}/billing/${key}/`}>
      {name}
    </ListLink>]
  ) : (
    []
  )
)}
```

or a filter-map:

```
{tabConfig
  .filter(({show}) => show(organization, isDisabled, subscription))
  .map(({key, name}) => (
    <ListLink key={key} to={`/settings/${organization.slug}/billing/${key}/`}>
      {name}
    </ListLink>
  ))
}
```

reduce can do _anything_, but when more concrete methods are available, I think we should prefer those.